xla/tests/int4_test.cc

```diff
@@ -157,6 +157,23 @@ XLA_TEST_F(HloTestBase, HorizontalLoopFusion) {
   EXPECT_TRUE(RunAndCompare(hlo_text, std::nullopt));
 }
 
+XLA_TEST_F(HloTestBase, ElementwiseOps) {
```

Comment:
Could we make an exhaustive test that tests each and every unary, binary, and ternary elementwise op?

We could have three parametrized test suites.

Reply:
Added exhaustive tests for ops that support integer arguments.